src/EventStore.Core/Cluster/MemberInfo.cs

```diff
@@ -59,15 +61,15 @@ public class MemberInfo : IEquatable<MemberInfo> {
 			int epochNumber,
 			Guid epochId,
 			int nodePriority,
-			bool isReadOnlyReplica) {
+			bool isReadOnlyReplica, string version = null) {
```

Comment:
This name should be kept consistent
```suggestion
			bool isReadOnlyReplica, string esVersion = null) {
```

Reply:
changed it